OpenRA.Mods.D2k/Activities/SwallowActor.cs

```diff
@@ -107,10 +107,14 @@ void PlayAttack(Actor self, WPos attackPosition, List<Player> affectedPlayers)
 			withSpriteBody.PlayCustomAnimation(self, sandworm.Info.MouthSequence);
 			Game.Sound.Play(swallow.Info.WormAttackSound, self.CenterPosition);
 
+			if (affectedPlayers.Count == 0)
+				return;
+
 			Game.RunAfterDelay(1000, () =>
 			{
-				foreach (var affectedPlayer in affectedPlayers)
-					NotifyPlayer(affectedPlayer, attackPosition);
+				if (Game.IsCurrentWorld(affectedPlayers[0].World))
```

Comment:
SwallowActor doesn't currently store `self` unless I add one.

Cast your votes for "least lame way to get a reference to the world". I'll implement whatever.